### PR TITLE
Readme and responsiveness

### LINKED ISSUE #29: Finalize README

### LINKED ISSUE #38: double-check responsiveness when we're at a stopping point

### PR DESCRIPTION
**Pull request by kristinmoon**
Mostly responsive... a couple issues:

I can't figure out how to get the logout button on the side-nav to function on click
the set size of the canvas is tricky. I can make the canvas responsive, but the elements on the board are not responsive. I left the canvas size as-is... looks funky at tablet and mobile phone size...
I also added a simple footer so when you get to the bottom of the page you know you're at the bottom. Feel free to add whatever in there if you're feeling it.

### CODE REVIEW

File: views/layouts/main.hbs [hunk 1]
```diff
@@ -51,6 +45,15 @@
 	<main>
 		{{{ body }}}
 	</main>
+
+	<footer class="page-footer">
+		<div class="footer-copyright">
+			<div class="container">
+				© 2021 CollabWhiteboard
+			</div>
+		</div>
```
Comment (joe-riley) [inline views/layouts/main.hbs:54]: I am not stopping this pr for this but the indents are a bit large.

**Review by joe-riley — APPROVED** [review]
I have one comment that shouldn't stop this merge. When in mobile mode teh login button is cut off on the modal. Fix it if you want. Or not :P

**Comment by joe-riley** [comment]
> Mostly responsive... a couple issues:
> 
> I can't figure out how to get the logout button on the side-nav to function on click

I am pretty sure I could fix the login stuff by changing the event handler for login to get a list of all of the login buttons and apply the event to all of them.
> the set size of the canvas is tricky. I can make the canvas responsive, but the elements on the board are not responsive. I left the canvas size as-is... looks funky at tablet and mobile phone size...

Yeah, the only thing I can think of is to have a horizontal scrollbar inside the canvas but then we get into pinch zooming and stuff to be able to get a feel for what the whole board looks like. I think that would be a nice to have but not needed at this time.
> I also added a simple footer so when you get to the bottom of the page you know you're at the bottom. Feel free to add whatever in there if you're feeling it.

Looks very professional. Nice add, as are all of the other changes you made. Thanks.

File: views/layouts/main.hbs (hunk 1 above)
Comment (kristinmoon) [reply views/layouts/main.hbs:54]: Very weird... definitely looks like unnecessary indentation here, but looks normal in VS code.

**Comment by kristinmoon** [comment]
> > Mostly responsive... a couple issues:
> > I can't figure out how to get the logout button on the side-nav to function on click
> 
> I am pretty sure I could fix the login stuff by changing the event handler for login to get a list of all of the login buttons and apply the event to all of them.

The other 3 buttons work in the side nav (login, signup, and profile)... it's just the logout that I can't get to function.

**Pull request merged**